tests/excel/test_range_specification_simple.py

```
@@ -0,0 +1,54 @@
+"""Phase 2: Range Specification機能の簡略化されたTDDテスト.
+
+Task 2.2: `:range:` オプション実装の未実装機能確認テスト
+"""
+
+import pytest
+
+
+def test_range_functionality_not_implemented():
+    """Range機能が未実装であることを確認するテスト（RED段階）。"""
```

Ruff (0.11.9)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)